calculatorApi1.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000
+
+
+app.get("/add/:num1/:num2", (req, res) => {
```

Comment:
Try console logging `req.params.num1` and `req.params.num2` to see how you can move forward